.maintain/config/bridge-darwinia-crab.toml

```diff
@@ -1,71 +1,14 @@
 [darwinia]
 endpoint = "wss://rpc.darwinia.network"
 signer = "//Alice"
-# Bundle/Auto/Custom
-runtime_version_mode = "Bundle"
```

Comment:
Do not need this config anymore?

Reply:
Yes, we don't need this now. subxt will auto fetch currently spec version when create new client
https://github.com/darwinia-network/subxt/blob/a90697ca2926dd698f5f26f62837b9d966be2ad8/subxt/src/client.rs#L129-L134